modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/DurationUtil.kt

```diff
@@ -0,0 +1,12 @@
+package au.com.shiftyjelly.pocketcasts.utils
+
+import java.util.Locale
+import kotlin.time.Duration
+
+fun Duration.toHhMmSs() = toComponents { hours, minutes, seconds, _ ->
+    if (hours == 0L) {
+        kotlin.String.format(Locale.ROOT, "%02d:%02d", minutes, seconds)
```

Comment:
Do you need to prefix it with Kotlin?

Reply:
No, probably got added when I moved the function to this file. I'll change it.

Reply:
Fixed in https://github.com/Automattic/pocket-casts-android/pull/2545/commits/b982224a851677083591d8950ecf8a6d4d20853e